Dv mproject

ghdud119 opened the pull request.

LeeHaiLim committed:
.

LeeHaiLim committed:
.

LeeHaiLim committed:
gui

LeeHaiLim committed:
dvm

LeeHaiLim committed:
finaltest

ghdud119 committed:
test

LeeHaiLim committed:
.

LeeHaiLim committed:
test

ghdud119 committed:
test

LeeHaiLim committed:
Merge branch 'DVMproject' of https://github.com/sys09270883/ctip into main

ghdud119 committed:
2021.05.10 3:27 송호영

yoonseopshin reviewed (commented):
컴파일이 되야 머지할 수 있습니다.

LeeHaiLim committed:
.

LeeHaiLim committed:
.

LeeHaiLim committed:
.

LeeHaiLim committed:
.

LeeHaiLim committed:
Delete DVMproject/DVM/src/DVM directory

delete

LeeHaiLim committed:
Delete DVMproject/DVM/src/GUI directory

delete

LeeHaiLim committed:
Add files via upload

add

LeeHaiLim committed:
test

sys09270883 commented:
`Title 1..*`, `item 0..*` 같은 표현들을 자바 문법에 맞게 써야 합니다. 예를 들어 `List<Title>`, `Title[]`과 같은 형식으로 쓰면 될 것 같습니다.

LeeHaiLim committed:
Add Timer

ghdud119 committed:
Merge branch 'DVMproject' of https://github.com/sys09270883/ctip into main

 Conflicts:
	src/main/java/DVM/Message_Queue.java

ghdud119 committed:
2021.05.11 3:46 송호영
메시지큐를 레빗mq를 활용하여 실질적으로 제 컴퓨터에 서버를 구축해서 브로커로 활용하는 코드로 만들어놓았습니다.
아직 완성본은 아니고 hello world만 왔다갔다하지만 일단 어느정도 틀이 잡힌거 같아 먼저 푸시해드립니다.

ghdud119 committed:
2021.05.11 3:46 송호영
메시지큐를 레빗mq를 활용하여 실질적으로 제 컴퓨터에 서버를 구축해서 브로커로 활용하는 코드로 만들어놓았습니다.
아직 완성본은 아니고 hello world만 왔다갔다하지만 일단 어느정도 틀이 잡힌거 같아 먼저 푸시해드립니다.

LeeHaiLim committed:
a

Haen1231 committed:
...

Haen1231 committed:
...

ghdud119 committed:
Merge branch 'DVMproject' of https://github.com/sys09270883/ctip into main

 Conflicts:
	DVMproject/DVM/DVM.iml
	DVMproject/DVM/src/Logic/Controller.java

Haen1231 committed:
...

ghdud119 committed:
Merge branch 'DVMproject' of https://github.com/sys09270883/ctip into main

Haen1231 committed:
...

ghdud119 committed:
2021.05.11 송호영
DVM, Message 클래서 최신화

Haen1231 committed:
...

ghdud119 committed:
Merge branch 'DVMproject' of https://github.com/sys09270883/ctip into main

 Conflicts:
	DVMproject/DVM/src/Logic/C_NumberManager.java

LeeHaiLim committed:
a

LeeHaiLim committed:
a

LeeHaiLim committed:
a

ghdud119 committed:
Merge branch 'DVMproject' of https://github.com/sys09270883/ctip into main

 Conflicts:
	DVMproject/DVM/src/Logic/Controller.java

ghdud119 committed:
..

ghdud119 committed:
Merge remote-tracking branch 'origin/DVMproject' into main

LeeHaiLim committed:
a

LeeHaiLim committed:
Merge branch 'DVMproject' of https://github.com/sys09270883/ctip into main

ghdud119 committed:
..

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
hailimelee

ghdud119 committed:
..

ghdud119 committed:
..

ghdud119 committed:
Merge branch 'DVMproject' of https://github.com/sys09270883/ctip into main

ghdud119 committed:
..

ghdud119 committed:
..

ghdud119 committed:
..

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
hailimelee

Haen1231 committed:
.

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
hailimelee

Haen1231 committed:
.

Haen1231 committed:
.

Haen1231 committed:
.

LeeHaiLim committed:
hailimelee

Haen1231 committed:
.

Haen1231 committed:
.

LeeHaiLim committed:
hailimelee

Haen1231 committed:
.

Haen1231 committed:
Merge branch 'DVMproject' of https://github.com/sys09270883/ctip

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
.

LeeHaiLim committed:
.

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
.

LeeHaiLim committed:
.

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
hailimelee

Haen1231 committed:
Merge branch 'DVMproject' of https://github.com/sys09270883/ctip

ghdud119 committed:
..

ghdud119 committed:
..

ghdud119 committed:
..

ghdud119 committed:
Merge remote-tracking branch 'origin/DVMproject' into main

ghdud119 committed:
..

Haen1231 committed:
.

Haen1231 committed:
Merge branch 'DVMproject' of https://github.com/sys09270883/ctip

Haen1231 committed:
.

LeeHaiLim committed:
.

ghdud119 committed:
..

LeeHaiLim committed:
.

LeeHaiLim committed:
Merge branch 'DVMproject' of https://github.com/sys09270883/ctip into dev/hailimlee

Haen1231 committed:
.

ghdud119 committed:
..

ghdud119 committed:
Merge branch 'DVMproject' of https://github.com/sys09270883/ctip into main

ghdud119 committed:
..

LeeHaiLim committed:
hailimelee

ghdud119 committed:
..

ghdud119 committed:
..

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
hailimelee

ghdud119 committed:
..

ghdud119 committed:
Merge branch 'DVMproject' of https://github.com/sys09270883/ctip into main

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
hailimelee

LeeHaiLim committed:
.

LeeHaiLim committed:
.

yoonseopshin committed:
Add project

LeeHaiLim committed:
Merge remote-tracking branch 'origin/feat/add-project' into dev/hailimlee

# Conflicts:
#	src/main/java/GUI/AddItemMenu.java
#	src/main/java/GUI/CardPayUI.java
#	src/main/java/GUI/DVMMenu.java
#	src/main/java/GUI/InfoCnumberErrUI.java
#	src/main/java/GUI/InfoNoItemUI.java
#	src/main/java/GUI/InfoReturnItemUI.java
#	src/main/java/GUI/InputLine.java
#	src/main/java/GUI/MainMenu.java
#	src/main/java/GUI/ManItemMenu.java
#	src/main/java/GUI/ManTitleMenu.java
#	src/main/java/GUI/PayErrUI.java
#	src/main/java/GUI/PaymentMenu.java
#	src/main/java/GUI/Sleep.java
#	src/main/java/GUI/SmartPayUI.java
#	src/main/java/Logic/C_Number.java
#	src/main/java/Logic/C_NumberManager.java
#	src/main/java/Logic/Controller.java
#	src/main/java/Logic/DVM.java
#	src/main/java/Logic/Item.java
#	src/main/java/Logic/Message.java
#	src/main/java/Logic/Message_Queue.java
#	src/main/java/Logic/Payment.java
#	src/main/java/Logic/Title.java

LeeHaiLim committed:
.
